### DISCUSSION TITLE
How to delete a newly created folder in wiki.js?

### DISCUSSION THREAD

**Discussion by myp9921** [requarks/wiki#6339] [Help / Questions]
How to delete a newly created folder in wiki.js?

**Comment by Keksuccino** [comment]
By folder, do you mean a page directory? Because that's just a pseudo-folder and not an actual one. It goes away when there are no pages in it anymore.

**Reply by myp9921** [reply]
When there were no files in the folder, it did not disappear and was eventually deleted from the database.
